In file included from ../../../home/z/build/1packages/chromium-dev-git/fetch_gclient_base/checkout_root/src/chrome/browser/extensions/browser_context_keyed_service_factories.cc:24:
../../../home/z/build/1packages/chromium-dev-git/fetch_gclient_base/checkout_root/src/chrome/browser/extensions/api/gcd_private/gcd_private_api.h:11:10: fatal error: 'chrome/common/extensions/api/gcd_private.h' file not found
#include "chrome/common/extensions/api/gcd_private.h"
         ^
1 error generated.

In file included from ../../../home/z/build/1packages/chromium-dev-git/fetch_gclient_base/checkout_root/src/chrome/browser/extensions/browser_context_keyed_service_factories.cc:29:
../../../home/z/build/1packages/chromium-dev-git/fetch_gclient_base/checkout_root/src/chrome/browser/extensions/api/mdns/mdns_api.h:17:10: fatal error: 'chrome/common/extensions/api/mdns.h' file not found
#include "chrome/common/extensions/api/mdns.h"
         ^
1 error generated.

../../../home/z/build/1packages/chromium-dev-git/fetch_gclient_base/checkout_root/src/chrome/browser/extensions/browser_context_keyed_service_factories.cc:97:15: error: no member named 'GcdPrivateAPI' in namespace 'extensions'
  extensions::GcdPrivateAPI::GetFactoryInstance();
  ~~~~~~~~~~~~^
../../../home/z/build/1packages/chromium-dev-git/fetch_gclient_base/checkout_root/src/chrome/browser/extensions/browser_context_keyed_service_factories.cc:113:15: error: no member named 'MDnsAPI' in namespace 'extensions'
  extensions::MDnsAPI::GetFactoryInstance();
  ~~~~~~~~~~~~^
2 errors generated.

diff --git a/chrome/browser/extensions/browser_context_keyed_service_factories.cc b/chrome/browser/extensions/browser_context_keyed_service_factories.cc
index 9b39773..1585e01 100644
--- a/chrome/browser/extensions/browser_context_keyed_service_factories.cc
+++ b/chrome/browser/extensions/browser_context_keyed_service_factories.cc
@@ -21,12 +21,12 @@
 #include "chrome/browser/extensions/api/extension_action/extension_action_api.h"
 #include "chrome/browser/extensions/api/feedback_private/feedback_private_api.h"
 #include "chrome/browser/extensions/api/font_settings/font_settings_api.h"
-#include "chrome/browser/extensions/api/gcd_private/gcd_private_api.h"
+//#include "chrome/browser/extensions/api/gcd_private/gcd_private_api.h"
 #include "chrome/browser/extensions/api/history/history_api.h"
 #include "chrome/browser/extensions/api/hotword_private/hotword_private_api.h"
 #include "chrome/browser/extensions/api/identity/identity_api.h"
 #include "chrome/browser/extensions/api/language_settings_private/language_settings_private_delegate_factory.h"
-#include "chrome/browser/extensions/api/mdns/mdns_api.h"
+//#include "chrome/browser/extensions/api/mdns/mdns_api.h"
 #include "chrome/browser/extensions/api/omnibox/omnibox_api.h"
 #include "chrome/browser/extensions/api/passwords_private/passwords_private_event_router_factory.h"
 #include "chrome/browser/extensions/api/preference/chrome_direct_setting_api.h"
@@ -94,7 +94,7 @@ void EnsureBrowserContextKeyedServiceFactoriesBuilt() {
   extensions::ExtensionWebUIOverrideRegistrar::GetFactoryInstance();
   extensions::FeedbackPrivateAPI::GetFactoryInstance();
   extensions::FontSettingsAPI::GetFactoryInstance();
-  extensions::GcdPrivateAPI::GetFactoryInstance();
+//  extensions::GcdPrivateAPI::GetFactoryInstance();
   extensions::HistoryAPI::GetFactoryInstance();
   extensions::HotwordPrivateEventService::GetFactoryInstance();
   extensions::IdentityAPI::GetFactoryInstance();
@@ -110,7 +110,7 @@ void EnsureBrowserContextKeyedServiceFactoriesBuilt() {
 #if defined(OS_CHROMEOS)
   extensions::LogPrivateAPI::GetFactoryInstance();
 #endif
-  extensions::MDnsAPI::GetFactoryInstance();
+//  extensions::MDnsAPI::GetFactoryInstance();
 #if defined(OS_CHROMEOS)
   extensions::MediaPlayerAPI::GetFactoryInstance();
 #endif
